Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
     <ItemGroup>
-        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net472" Version="1.0.2"/>
+        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net472" Version="1.0.3"/>
```

Comment:
Could you tell if this change was required for something, or is it just to have a more recent version?
I wonder if I need to update it in our plugin too. 🙂

Reply:
Without this, compilation failed, because we have NuGet warnings upgraded to errors somewhere (and Rider uses 1.0.3 version of this package).

I think it's safe to ignore refasm update, unless you have compilation issues.

Reply:
Thanks, I'll update it just to make it easier later when we can use F# plugin in the solution.